mpl_interactions/generic.py

```diff
@@ -1,21 +1,21 @@
+from collections.abc import Callable, Iterable
+
+import matplotlib.cm as cm
 import numpy as np
 from matplotlib import __version__ as mpl_version
 from matplotlib import get_backend
+from matplotlib.colors import TABLEAU_COLORS, XKCD_COLORS, to_rgba_array
 from matplotlib.path import Path
 from matplotlib.pyplot import close, subplots
 from matplotlib.widgets import LassoSelector
 from numpy import asanyarray, asarray, max, min, swapaxes
 from packaging import version
 
+from .helpers import *
 from .utils import figure, ioff, nearest_idx
 
 # functions that are methods
-__all__ = [
-    "heatmap_slicer",
-    "zoom_factory",
-    "panhandler",
-    "image_segmenter",
-]
+__all__ = ["heatmap_slicer", "zoom_factory", "panhandler", "image_segmenter", "hyperslicer"]
```

Comment:
add a comma after `hyperslicer` so that black will explode the list.

Reply:
adding the comma implies that more things may be added in the future. and when you have a trailing comma it makes teh diff for adding a new line only change one line instead of two